ausonius-editions.csl

```diff
@@ -134,7 +139,7 @@
       <if is-numeric="edition">
         <group delimiter=", ">
```

Comment:
The comma should be removed, in order to avoid something like "2e, éd." for a second edition